modules/era/sql/era_droplist.sql

```diff
@@ -9172,19 +9169,20 @@ INSERT INTO `mob_droplist` VALUES (1105,0,0,1000,12985,@VRARE);   -- Holly Clogs
 INSERT INTO `mob_droplist` VALUES (1105,2,0,1000,750,0);          -- Silver Beastcoin (Steal)
 
 -- ZoneID: 190 - Goblin Leecher
-INSERT INTO `mob_droplist` VALUES (1106,0,0,1000,2757,@UNCOMMON); -- Orcish Armor Plate (Uncommon, 10%)
-INSERT INTO `mob_droplist` VALUES (1106,0,0,1000,4666,@RARE);     -- Scroll Of Paralyze (Rare, 5%)
-INSERT INTO `mob_droplist` VALUES (1106,0,0,1000,4733,@RARE);     -- Scroll Of Protectra (Rare, 5%)
-INSERT INTO `mob_droplist` VALUES (1106,0,0,1000,4744,@RARE);     -- Scroll Of Invisible (Rare, 5%)
+INSERT INTO `mob_droplist` VALUES (1106,0,0,1000,4666,@UNCOMMON); -- Scroll Of Paralyze (Rare, 5%)
+INSERT INTO `mob_droplist` VALUES (1106,0,0,1000,4680,@UNCOMMON); -- Scroll Of Barsleep (Very Rare, 1%)
+INSERT INTO `mob_droplist` VALUES (1106,0,0,1000,2757,@RARE);     -- Orcish Armor Plate (Rare, 5%)
 INSERT INTO `mob_droplist` VALUES (1106,0,0,1000,4745,@RARE);     -- Scroll Of Sneak (Rare, 5%)
-INSERT INTO `mob_droplist` VALUES (1106,0,0,1000,4746,@RARE);     -- Scroll Of Deodorize (Rare, 5%)
-INSERT INTO `mob_droplist` VALUES (1106,0,0,1000,4667,@VRARE);    -- Scroll Of Silence (Very Rare, 1%)
-INSERT INTO `mob_droplist` VALUES (1106,0,0,1000,4680,@VRARE);    -- Scroll Of Barsleep (Very Rare, 1%)
+INSERT INTO `mob_droplist` VALUES (1106,0,0,1000,4667,@RARE);     -- Scroll Of Silence (Rare, 5%)
+INSERT INTO `mob_droplist` VALUES (1106,0,0,1000,4733,@RARE);     -- Scroll Of Protectra (Rare, 5%)
+INSERT INTO `mob_droplist` VALUES (1106,0,0,1000,4744,@VRARE);    -- Scroll Of Invisible (Very Rare, 1%)
+INSERT INTO `mob_droplist` VALUES (1106,0,0,1000,4746,@VRARE);    -- Scroll Of Deodorize (Very Rare, 1%)
 INSERT INTO `mob_droplist` VALUES (1106,0,0,1000,4681,@VRARE);    -- Scroll Of Barpoison (Very Rare, 1%)
 INSERT INTO `mob_droplist` VALUES (1106,0,0,1000,4683,@VRARE);    -- Scroll Of Barblind (Very Rare, 1%)
 INSERT INTO `mob_droplist` VALUES (1106,0,0,1000,12473,@VRARE);   -- Poets Circlet (Very Rare, 1%)
 INSERT INTO `mob_droplist` VALUES (1106,0,0,1000,12729,@VRARE);   -- Linen Cuffs (Very Rare, 1%)
 INSERT INTO `mob_droplist` VALUES (1106,0,0,1000,12857,@VRARE);   -- Linen Slops (Very Rare, 1%)
+INSERT INTO `mob_droplist` VALUES (1106,0,0,1000,12985,@VRARE);   -- Holly Clogs (Very Rare, 1%)
```

Comment:
According to the wiki some of these drop rates seem to be extremely low. Paralyze and Barsleep both took to be 10% TH0 (Common) http://www.ffxidb.com/zones/190/goblin-leecher

Reply:
10% is Uncommon, 15% is Common. Rechecking, everything looks to match up correctly already.

Reply:
Sorry meant to say uncommon!